src/components/FindMovie/FindMovie.tsx

```diff
@@ -16,42 +95,63 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className={classNames(
+                'input',
+                {
+                  'is-danger': movieFoundError,
+                },
+              )}
+              value={queryTitle}
+              onChange={handleInputChange}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {movieFoundError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                {
+                  'is-loading': isLoading,
+                },
+              )}
+              disabled={queryTitle.trim().length === 0}
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable